service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/AdditionalApplicationsBundle.java

```diff
@@ -37,4 +39,13 @@ public String toLabel() {
 
         throw new MissingApplicationException(uploadedDateTime);
     }
+
+    public YesNo getApplicationReviewed() {
+        if (applicationReviewed == null) {
+            // DFPL-1047 Reviewing is not required for documents uploaded in historic cases
+            return YesNo.YES;
+        } else {
+            return applicationReviewed;
+        }
+    }
```

Comment:
Can be simplified as `Optional.ofNullable(applicationReviewed).orElse(YesNo.YES)`?